code-challenges.js

```diff
@@ -39,3 +108,18 @@ var numbersToAdd2 = [0, 7, -8, 12]
 // Expected output: [0, 7, -1, 11]
 var numbersToAdd3 = []
 // Expected output: []
+
+// create function that takes array as an argument
+const accumArray = (array) => {
+  // create a variable to hold the result of the addition
+  let sum = 0
+  // loop through array and add result to new value
+  let mappedArray = array.map((value) => {
+    return (sum = sum + value)
+  })
+  // return the array of the accumilated sum
+  return mappedArray
+}
+console.log(accumArray(numbersToAdd1))
+console.log(accumArray(numbersToAdd2))
+console.log(accumArray(numbersToAdd3))
```

Comment:
nice work in this section. I appreciate you commenting your code to show that you understood what was happening on each line.